solutions/fully-configurable/scripts/delete_secrets.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
```

Comment:
I would leave a code comment which shows the script usage, as well as the dependencies (curl and jq)